20231114_external-mass-tracers/README.md

```diff
@@ -0,0 +1,53 @@
+LiteBIRD external mass tracers
+=======================================
+
+## Summary
+We provide 500 realizations of the harmonic coefficients of the external mass tracers (signal/noise). 
+For a reference, we also save their signal auto/cross spectra which are used to generate the harmonic coefficients. 
+
+## Description of simulation
+For the LiteBIRD delensing study, in addition to the S4 CMB lensing map, we plan to explore the importance of the external mass tracers, i.e., 
+the CIB and galaxies. In this project, we follow the strategy of https://arxiv.org/abs/2110.09730 where we simply generate these external tracers 
+as Gaussian random fields drawn from auto and cross-angular power spectra between CMB-lensing, CIB, and galaxies, with the Gaussian kappa provided in the above. 
+
+ * The plan here is to assume a single CIB map that mimics the Planck GNILC CIB map over ~ 40% of the sky. CIB is simply the post-component-separated map and 
```

Comment:
how does this CIB compare to the PySM CIB used in https://github.com/litebird/litebirdXS4/blob/master/20230810_foregrounds_cmb/README.md?